projects/step-frontend/src/lib/modules/plan/components/plan-selection/plan-selection.component.scss

```diff
@@ -0,0 +1,4 @@
+.mat-column-singleSelect,
+.mat-column-bulkSelection {
+  width: 48px;
```

Comment:
I see this in a 2nd PR, does it make sense to have it as a global style?

Reply:
Even if it duplicates code I would leave it here since it directly influences how the table is rendered.